libs/ui-toolkit/src/components/dialog/dialog.tsx

```diff
@@ -0,0 +1,52 @@
+import * as DialogPrimitives from '@radix-ui/react-dialog';
+import classNames from 'classnames';
+import { ReactNode } from 'react';
+import { Intent } from '../../utils/intent';
+import { Icon } from '../icon';
+
+interface DialogProps {
+  children: ReactNode;
+  open: boolean;
+  onChange: (isOpen: boolean) => void;
+  title?: string;
+  intent?: Intent;
+}
+
+export function Dialog({
+  children,
+  open,
+  onChange,
+  title,
+  intent,
+}: DialogProps) {
+  const contentClasses = classNames(
+    // Positions the modal in the center of screen
+    'fixed w-[520px] px-28 py-24 top-[50%] left-[50%] translate-x-[-50%] translate-y-[-50%]',
+    // Need to apply background and text colors again as content is rendered in a portal
+    'dark:bg-black dark:text-white-95 bg-white text-black-95',
+    // For some reason if I use getIntentShadow from utils/intent the styles arent applied
+    'shadow-callout',
+    {
+      'shadow-intent-danger': intent === Intent.Danger,
```

Comment:
Can we use the intent util for this?

Reply:
Or even more. We should use `<Callout>` component

Reply:
Switched to using intent shadow helper (which is now working)